### DISCUSSION TITLE
Is there any assumption on the maximum length of the client command?

### DISCUSSION THREAD

**Discussion by antony-jeong** [ANLAB-KAIST/KENSv3#36] [Project Q&A 1. Application Layer (Socket)]
Is there any assumption on the maximum length of the client command?

I want to use static buffer, not dynamic memory allocation,
but if the length of the request exceeds the buffer,
I cannot submit the request from the client using submitAnswer() because I have to submit at once.

**Comment by Pusnow** [comment]
No, there is no assumption. Your client should handle arbitrary length command.
But as long as you pass all test cases, it's okay.
